crypto/fips/fips_kdf_selftest.c

```diff
@@ -361,7 +364,9 @@ static int FIPS_selftest_krb5kdf(void)
     EVP_KDF_CTX_free(kctx);
     return ret;
 }
+#endif
 
+#if 0
```

Comment:
@jallisonciq this is single step KDF or one-step this should be approved

Reply:
These aren't removed for algorithm completeness, but to remove an "unused but defined function" warning. These are static functions that are not referenced *anywhere* else in this file. So they can be commented out to remove the warning.